src/main/java/uk/gov/hmcts/reform/sscs/service/DwpAddressLookup.java

```diff
@@ -16,6 +22,37 @@
 
     private static final String PIP = "PIP";
     private static final String ESA = "ESA";
+    private static final String EXCELA = "excela";
+    private static final String ADDRESS = "address";
+    private static final String LINE_1 = "line1";
+    private static final String LINE_2 = "line2";
+    private static final String LINE_3 = "line3";
+    private static final String POST_CODE = "postCode";
+    private static final String CODE = "code";
+
+    private static JSONObject configObject;
+
+    static {
+        JSONParser parser = new JSONParser();
+        try {
+            configObject = (JSONObject) parser.parse(
+                Files.newBufferedReader(Paths.get(
+                    Objects.requireNonNull(Thread.currentThread().getContextClassLoader().getResource("dwpAddresses.json")).toURI())
+                )
+            );
+        } catch (Exception exception) {
+            log.error("Cannot parse dwp addresses.", exception);
+            throw new RuntimeException("cannot parse dwp addresses", exception);
+        }
+    }
+
+    private static final JSONObject EXCELA_CONFIG = (JSONObject) configObject.get(EXCELA);
```

Comment:
Guess this is fine, but wondered if there is a generic way of parsing each config? Potentially we could end up with a few more benefit types in here and it would require code to be copy and pasted.